### ISSUE TITLE
Dude!   Have you seen this?

### ISSUE THREAD

**Issue by FlaMike** [rstrouse/ESPSomfy-RTS#64]
Y'all are getting famous world-wide!  Too bad I can't understand a word he says.  I did just turn on translate to English subtitles & will watch it.

https://www.youtube.com/watch?v=dtvTQM6dqcQ

**Comment by rstrouse** [comment]
I listened to it in German.  I was waiting to hear the word dummkopf but it never came up.  If you are reading this @smart-live.net, thanks for the compliments!  If the video wasn't flattering I'll just go on believing it was.

**Comment by FlaMike** [comment]
Whatever translation errors or omissions there may have been I still think
you're a rock star!  I don't think you should ever close this "issue!". 😅

**Comment by caibo84** [comment]
Since I am from Germany, I can tell you that he did not say Dummkopf.  It was more in the direction that you are a genius. 

From my side thanks for this great solution. One year ago i made a controll based on the Arduino SOmfy libraries and integrated the blinds in IOBroker. I tried o also detect the level with the some scripts but it didn't work out well. I also had the problem when somebody using normal remote the level doesn't fit anymore.  you're solution with detecting the remote signals is really cool

**Comment by rstrouse** [comment]
Thanks for the confirmation but I don't know about the genius thing I do plenty of stupid things.  That being said, the receive portion of this almost had me buying equipment to listen for OOK pulses.  Who does that kind of thing?  In the end though, I am glad others have found utility in it.  

Really I wrote this because I hated that I didn't have control over my roller shades and dug into it deeper because there were screws that needed turning on it.  I almost didn't make the repo public.

**Comment by pfak** [comment]
Just got Somfy awning, and I see 4 days ago you added support. Thanks a lot, this project is great!

**Comment by rstrouse** [comment]
> Just got Somfy awning, and I see 4 days ago you added support. Thanks a lot, this project is great!

Aren't you glad you squinted for 4 more days?  Thanks for the kudos!

**Comment by FlaMike** [comment]
FYI, Mr. Kieser (pfak above) is my discord buddy who was interested in your project when I mentioned it a couple of months ago.  So happy everything worked out, including "just in time" awning features going GA!  :-)

**Comment by nardev** [comment]
Why the h you don't have some kind of a donate button.

You just saved me bunch of money and time.

**Comment by rstrouse** [comment]
@nardev just glad you found some use for it!

**Comment by piv3rt** [comment]
A big thank you to @rstrouse and all the other contributors to this great project! Impressive work! 👏
This works flawlessly in my home with HomeAssistant and inspired me to publish a [YouTube video](https://www.youtube.com/watch?v=1C-lgoN1hmk) too.

**Comment by rstrouse** [comment]
Great work on the video and great explanations of the operation.   I will look at adding a section to the wiki or perhaps setting up the discussions section so others can showcase their work.  Bravo!

**Comment by Nickduino** [comment]
> I almost didn't make the repo public.

That would have been a shame. Any other gem you're sitting on and have not made public?


Is it possible to do a batch change of the "simulated position" instead of editing all 16 (in my case) shutters?
No biggie if it's impossible, of course.

**Comment by kieranc** [comment]
> Why the h you don't have some kind of a donate button.
> 
> You just saved me bunch of money and time.

I'd also like to be able to buy you a beer @rstrouse this is a fantastic bit of software, well documented, easy to use and massively cheaper than the 200€ box somfy want to sell me to do the same job. Thank you!

**Comment by Nickduino** [comment]
> 200€ box somfy want to sell me to do the same job.

Does it do the same job? I.e. does it try to deduce the position of RTS shutters from every command sent to it?

**Comment by kieranc** [comment]
No, you're right, it doesn't do that, but in defence of the Somfy box, I feel like this project doesn't even try to monetize my personal info? Missed opportunities....

**Comment by Nickduino** [comment]
I wonder if the Tahoma does that. Like, do you see ads for umbrellas if your Somfy rain detector is active more often than the average customer? Hair gel if it's windy? Sunscreen if it's sunny? Coats if it's cold?

**Comment by dietrmat** [comment]
Thanks a lot for this awesome project @rstrouse 🤟 
I bought all components, connected all together and flashed an ESP32 for the first time.
Apart from that it is working perfectly with my shutters, this was so much fun. 😍 
I really like your documentation, it's detailed and fun.

**Comment by indus** [comment]
> this was so much fun

Exactly what I wanted to say, too. It’s the perfect sweet spot — a seemingly complex task made so approachable and smooth by your work that it was genuinely enjoyable. Thank you!

**Comment by bamje** [comment]
This project is absolutely amazing. Thank you, you are awesome
